src/pybind/mgr/telemetry/module.py

```diff
@@ -146,6 +147,13 @@ class Collection(str, enum.Enum):
         "channel": "basic",
         "nag": False
     },
+    {
+        "name": Collection.basic_stretch_cluster,
+        "description": "Stretch mode information for stretch clusters",
+        "channel": "basic",
+        # ASK: When do we nag the users?
+        "nag": False
```

Comment:
Question: When do we nag users? 

IIUC, we only do that for major data changes. What would constitute as major data changes? My assumption here for not nagging the user was that `stretch_cluster` is a small additive change and is not a sensitive information so it's okay to not nag the user. Please correct me if my understanding is wrong,

Reply:
Yes, you are correct with this assumption. We nagged when we added the perf channel since it was a bigger addition, but adding a smaller collection is usually not something we want to nag about. It would be good to confirm this with @yaarith though.

Reply:
Also, make sure to remove the comment.

Reply:
thanks for the clarification! have now removed the comment